pagopa/pn-frontend
# Download notification documents bff api - pn-10889

## Short description
Migrated the delivery and delivery push api to the bff ones for the download of the notification documents

## List of changes proposed in this pull request (PA/PF/PG)
- Auto generated client
- Removed methods from api layer (Notifications.api.ts)
- Updated redux layer using the auto generated api client
- Updated the NotificationDetail page and components
- Removed the management of the downloads through redux state
- Updated tests

## How to test (PA/PF/PG)
- Login and go to the notification detail -> try to download the documents in "Documenti allegati", "AVVISO DI AVVENUTA RICEZIONE" and in the timeline
- Do the first check for multi recipient notifications
- Do the first check for a notification oldest than 120 days and check that the retryAfter alert is shown
- Do the first check with a delegated user
